{"content": {"suites": {"0": {"status": {"total_pass": 1, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 0, "total_error": 0}, "tests": {"0": {"status": "PASS", "message": "", "test_name": "test_product_creation", "rerun": "0"}}, "suite_name": "backend/tests/api/test_products.py"}, "1": {"suite_name": "backend/tests/api/test_user.py", "tests": {"0": {"status": "PASS", "message": "", "test_name": "test_user_create", "rerun": "0"}, "1": {"status": "PASS", "message": "", "test_name": "test_set_check_password", "rerun": "0"}, "2": {"status": "PASS", "message": "", "test_name": "test_register_user", "rerun": "0"}}, "status": {"total_pass": 3, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 0, "total_error": 0}}, "2": {"suite_name": "backend/tests/unit/test_unit.py", "tests": {"0": {"status": "PASS", "message": "", "test_name": "test_product_model", "rerun": "0"}, "1": {"status": "PASS", "message": "", "test_name": "test_review_model", "rerun": "0"}, "2": {"status": "PASS", "message": "", "test_name": "test_order_model", "rerun": "0"}, "3": {"status": "PASS", "message": "", "test_name": "test_order_item_model", "rerun": "0"}, "4": {"status": "PASS", "message": "", "test_name": "test_shipping_address_model", "rerun": "0"}}, "status": {"total_pass": 5, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 0, "total_error": 0}}, "3": {"suite_name": "backend/tests/api/test_products.py", "tests": {"0": {"status": "FAIL", "message": "   assert 401 == 200\n    +  where 401 = <Response status_code=401, \"application/json\">.status_code\n", "test_name": "test_api_product_creation", "rerun": "0"}}, "status": {"total_pass": 0, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 1, "total_error": 0}}, "4": {"suite_name": "backend/tests/integration/test_integration.py", "tests": {"0": {"status": "FAIL", "message": "       TypeError: int() argument must be a string, a bytes-like object or a real number, not 'NoneType'\n       django.core.paginator.PageNotAnInteger: That page number is not an integer\n   RuntimeError: Database access not allowed, use the \"django_db\" mark, or the \"db\" or \"transactional_db\" fixtures to enable it.\n", "test_name": "test_api_product_list", "rerun": "0"}, "1": {"status": "FAIL", "message": "   RuntimeError: Database access not allowed, use the \"django_db\" mark, or the \"db\" or \"transactional_db\" fixtures to enable it.\n", "test_name": "test_api_product_retrieve", "rerun": "0"}, "2": {"status": "FAIL", "message": "   assert 405 == 200\n    +  where 405 = <Response status_code=405, \"application/json\">.status_code\n", "test_name": "test_api_product_update", "rerun": "0"}, "3": {"status": "FAIL", "message": "   RuntimeError: Database access not allowed, use the \"django_db\" mark, or the \"db\" or \"transactional_db\" fixtures to enable it.\n", "test_name": "test_api_product_delete", "rerun": "0"}, "4": {"status": "FAIL", "message": "   assert 401 == 400\n    +  where 401 = <Response status_code=401, \"application/json\">.status_code\n", "test_name": "test_api_product_creation_invalid", "rerun": "0"}}, "status": {"total_pass": 0, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 5, "total_error": 0}}}}, "date": "January 12, 2025", "start_time": 1736674161.6352346, "total_suite": 5, "status": "FAIL", "status_list": {"pass": "9", "fail": "6", "skip": "0", "error": "0", "xpass": "0", "xfail": "0", "rerun": "0"}, "total_tests": "15"}